### ISSUE TITLE
Undefined index: user 

### ISSUE THREAD

**Issue by vkamelin** [postaddictme/instagram-php-scraper#285]
Notice: Undefined index: user in /.../vendor/raiym/instagram-php-scraper/src/InstagramScraper/Instagram.php on line 237

```
$instagram = new Instagram();
$posts = $instagram->getMedias($setting['account'], 8);
```

**Comment by lon9man** [comment]
+1

**Comment by gsound** [comment]
Yes I am having the same issue.
It now looks like this.

```
Array
(
    [logging_page_id] => profilePage_37396583
    [show_suggested_profiles] => 
    [graphql] => Array
        (
            [user] => Array
                (
                    [biography] => 
                    [blocked_by_viewer] => 
                    [country_block] => 
//...
```

**Comment by rdesorme** [comment]
**How to fix** 
In src/InstagramScraper/Instagram.php do some replacement in this order:
`$arr['user']['media']`
=> `$arr['graphql']['user']['edge_owner_to_timeline_media']`
`$arr['graphql']['user']['edge_owner_to_timeline_media']['nodes']`
=>  `$arr['graphql']['user']['edge_owner_to_timeline_media']['edges']`
`Media::create($mediaArray)`
=> `Media::create($mediaArray['node'])`

`getPaginateMedias` isn't working with this, paginate has change. [Here an helpfull post](https://stackoverflow.com/questions/49265013/instagram-a-1max-id-end-cursor-isnt-working-for-public-user-feeds#answer-49266320)

**Comment by ZipDriver** [comment]
Looks like data structure from Instagram changed.

**Comment by gsound** [comment]
This is fixed with #290

**Comment by brunamoreirav** [comment]
> Aviso: Índice indefinido: usuário em /.../vendor/raiym/instagram-php-scraper/src/InstagramScraper/Instagram.php na linha 237
> 
> ```
> $instagram = new Instagram();
> $posts = $instagram->getMedias($setting['account'], 8);
> ```
